test/general/header_inclusion_order_async_1.pass.cpp

```diff
@@ -34,7 +34,7 @@ main()
     std::vector<T, allocator> data(n, 1, alloc);
 
     auto f = oneapi::dpl::experimental::reduce_async(TestUtils::make_device_policy(q), data.begin(), data.end());
-    f.wait();
+    f.deferrable_wait();
```

Comment:
Same as above, users probably should call `wait()`

Reply:
@akukanov how do you think, which call we should use in this case?

Reply:
Fixed: restored `wait()` call.